response_operations_ui/contexts/collection_exercise.py

```diff
@@ -0,0 +1,265 @@
+from flask import url_for
+
+REMINDER_EMAILS = {
+    "reminder": "First reminder",
+    "reminder2": "Second reminder",
+    "reminder3": "Third reminder",
+    "reminder4": "Fourth reminder",
+    "reminder5": "Fifth reminder",
+}
+
+NUDGE_EMAILS = {
+    "nudge_email_0": "First nudge email",
+    "nudge_email_1": "Second nudge email",
+    "nudge_email_2": "Third nudge email",
+    "nudge_email_3": "Fourth nudge email",
+    "nudge_email_4": "Fifth nudge email",
+}
+
+CI_TABLE_LINK_TEXT = {
+    "EQ": {"no_instrument": "Add", "restricted": "View", "has_permission": "Select or Add"},
```

Comment:
I realise this bit is copied over, but when the user does not have survey edit permissions, they cannot add, upload or modidy EQ or SEFT instruments, so it showing Add and Upload where there are no instuments indicates to the user they would be able to do that when they are not. This probably seems like another card but I've just noticed it

Reply:
I would agree, definitely a bug, but also not connected to this PR.